src/toolchain/export-mac-and-ios.md

```diff
@@ -0,0 +1,263 @@
+
+
+Edit the project's `.gdextension` file to include support for iOS.
+This file will probably be at `godot/{YourCrate}.gdextension`.
+The format will be similar to the following:
+
+```ini
+[libraries]
+...
+ios.release = "res://../rust/target/release/libmy_lib.ios.framework"
+```
+
+## Code Signing and Notarizing (macOS only)
+
+Note: This step is only needed if you want to share the library. If you are building the whole game, you will sign everything and don't need to sign the library. You can skip this step.
+
+In order to Code Sign and Notarize your app, you will first need to gather some stuff from your Apple Developer Enrolled Account. We will put them each in an environment variable and use a script to sign, so it's easier to run. Here are the environment variables needed:
+
+- APPLE_CERT_BASE64
+- APPLE_CERT_PASSWORD
+- APPLE_DEV_ID
+- APPLE_DEV_TEAM_ID
+- APPLE_DEV_PASSWORD
+- APPLE_DEV_APP_ID
+
+Firstly, make sure to enroll your Apple ID to Developer Program:
+
+- Create an Apple ID if you don’t have one already.
+- Use your Apple ID to register in the Apple Developer Program by going to [developer.apple.com](https://developer.apple.com)
+- Accept all agreements from the Apple Developer Page.
+
+### APPLE_DEV_ID - Apple ID
+
+Your email used for your Apple ID.
+```
+APPLE_DEV_ID = email@provider.com
+```
+### APPLE_DEV_TEAM_ID - Apple Team ID
+
+Go to [developer.apple.com](https://developer.apple.com). Go to account.
+
+Go to membership details. Copy Team ID.
+```
+APPLE_DEV_TEAM_ID = 1ABCD23EFG
+```
+### APPLE_DEV_PASSWORD - Apple App-Specific Password
+
+Create Apple App-Specific Password. Copy the password.
+```
+APPLE_DEV_PASSWORD = abcd-abcd-abcd-abcd
+```
+
+### APPLE_CERT_BASE64 and APPLE_CERT_PASSWORD and APPLE_DEV_APP_ID
+
+Go to [developer.apple.com](https://developer.apple.com). Go to account.
+
+Go to certificates.
+
+Click on + at Certificates tab. Create Developer ID Application. Click Continue.
+
+Leave profile type as is. Create a certificate signing request from a mac. You can use your own name and email address. Save the file to disk. You will get a file called CertificateSigningRequest.certSigningRequest. Upload it to the Developer ID Application request. Click Continue.
+
+Download the certificate. You will get a file developerID_application.cer.
+
+On a Mac, right click and select open. Add it to the login keychain. In the Keychain Access app that opened, login Keychain tab, go to Keys, sort by date modified, expand your key (the key should have name you entered at common name Common Name), right click the expanded certificate, get info, and copy the text at Details -> Subject Name -> Common Name. Eg.
+```
+APPLE_DEV_APP_ID = Developer ID Application: Common Name (1ABCD23EFG)
+```
+Then, select the certificate, right click and click export. At file format select p12. When exporting, set a password for the certificate. This will be APPLE_CERT_PASSWORD. You will get a Certificates.p12 file.
```

Comment:
```suggestion
Then, select the certificate, right click and click export. At file format select `p12`. When exporting, set a password for the certificate. This will be the value of `APPLE_CERT_PASSWORD`. You will get a `Certificates.p12` file.
```